inc/saiacl.h

```diff
@@ -1419,6 +1419,44 @@ typedef enum _sai_acl_table_attr_t
      */
     SAI_ACL_TABLE_ATTR_AVAILABLE_ACL_COUNTER,
 
+    /**
+     * @brief Start of Table Match valid bits
+     *
+     * The valid bits specify the bits of match field that should be
+     * included in the lookup key. If a match field does not have
+     * valid bits specified, all bits in the field are valid.
+     *
+     * For tables implemented using Exact Match, there is no further
+     * key masking supported in table entries. The mask for such
+     * entries needs to be set to all 1s.
+     */
+    SAI_ACL_TABLE_ATTR_FIELD_VALID_BITS_START = 0x00003000,
+
+    /**
+     * @brief Src IPv6 Valid bits
+     *
+     * @type sai_acl_field_data_mask_t sai_ip6_t
+     * @flags CREATE_ONLY
+     * @default ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff
+     * @validonly SAI_ACL_TABLE_ATTR_FIELD_SRC_IPV6 == true
```

Comment:
I believe this is only for all match type
SAI_ACL_MATCH_TYPE_TERNARY_AND_EXACT
and
SAI_ACL_MATCH_TYPE_EXACT.

Add a valid only condition

Reply:
> I believe this is only for all match type SAI_ACL_MATCH_TYPE_TERNARY_AND_EXACT and SAI_ACL_MATCH_TYPE_EXACT.
> 
> Add a valid only condition

This is valid for SAI_ACL_MATCH_TYPE_TERNARY also as it allows us to reduce the tcam key width